### PR TITLE
Migration to Py3/Qt5

### PR DESCRIPTION
**Pull request by elpaso**

### CODE REVIEW

**Comment by nyalldawson** [comment]
(For reference I will initially review this!)

**Comment by nyalldawson** [comment]
@elpaso can you double-check 4076dd6  please?

**Comment by elpaso** [comment]
@nyalldawson https://github.com/inasafe/parameters/commit/4076dd69d6b2429124403eca8322b2acdfb4a918  looks good to me.

**Comment by nyalldawson** [comment]
@timlinux This is ready for merging, when the time is right for you.

File: parameters/qt_widgets/input_list_parameter_widget.py [hunk 1]
```diff
@@ -1,9 +1,14 @@
+from builtins import str
 # coding=utf-8
 
-from PyQt4.QtCore import Qt
-from PyQt4.QtGui import (
-    QListWidget, QAbstractItemView, QLineEdit, QVBoxLayout, QHBoxLayout,
-    QPushButton, QListWidgetItem)
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import (QListWidget,
```
Comment (timlinux) [inline parameters/qt_widgets/input_list_parameter_widget.py]: We should pull these indents left

File: parameters/qt_widgets/list_parameter_widget.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
 # coding=utf-8
 
-from PyQt4.QtCore import Qt
-from PyQt4.QtGui import (
-    QListWidget,
-    QAbstractItemView,
-    QListWidgetItem,
-    QVBoxLayout)
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import (QListWidget,
```
Comment (timlinux) [inline parameters/qt_widgets/list_parameter_widget.py]: Pull left

File: parameters/qt_widgets/numeric_parameter_widget.py [hunk 3]
```diff
@@ -2,8 +2,11 @@
 """Numeric Parameter Widget"""
 
 # noinspection PyPackageRequirements
-from PyQt4.QtCore import Qt
-from PyQt4.QtGui import QLabel, QSizePolicy, QWidget, QComboBox
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import (QLabel,
```
Comment (timlinux) [inline parameters/qt_widgets/numeric_parameter_widget.py]: Pull left

**Comment by timlinux** [comment]
If you can just pull those imports left we can merge, thanks!

**Pull request merged**